data_diff/diff_tables.py

```diff
@@ -68,27 +71,47 @@ def get_values(self) -> list:
         select = self._make_select(columns=self._relevant_columns)
         return self.database.query(select, List[Tuple])
 
-    def choose_checkpoints(self, count: int) -> List[DbKey]:
-        "Suggests a bunch of evenly-spaced checkpoints to split by"
-        ratio = int(self.count / count)
-        assert ratio > 1
-        skip = f"mod(idx, {ratio}) = 0"
-        select = self._make_select(table=Enum(self.table_path, order_by=self.key_column), where=skip)
-        return self.database.query(select, List[int])
+    def choose_checkpoints(self, bisection_factor: int) -> List[DbKey]:
+        """
+        Choose `bisection_factor - 1` (because of start and end) checkpoints in
+        the keyspace.
 
-    def find_checkpoints(self, checkpoints: List[DbKey]) -> List[DbKey]:
```

Comment:
We completely get rid of `find_checkpoints` since we're just creating these segments. It doesn't matter if the ids actually exist